echopype/metrics/summary_statistics.py

```diff
@@ -0,0 +1,56 @@
+"""
+"echo metrics" functions
+Reference:
+Urmy et al. 2012. Measuring the vertical distributional variability of pelagic fauna in Monterey Bay.
+ICES Journal of Marine Science 69 (2): 184-196.
+http://icesjms.oxfordjournals.org/cgi/content/full/fsr205?ijkey=tSU0noNUWz4bj57&keytype=ref
+Implementation:
+https://github.com/ElOceanografo/EchoMetrics/blob/master/echometrics/echometrics.py
+"""
+
+import numpy as np
+
+
+def delta_z(ds, zname='range'):
+    if zname not in ds:
+        raise ValueError(f"{zname} not in the input Dataset!")
+    dz = ds[zname].diff(dim='range_bin')  # here assume range or depth are data variables with the same dims as Sv
+    return dz.where(dz != 0, np.nan)
+
+
+def convert_to_linear(da_Sv):
+    return 10 ** (da_Sv / 10)  # convert Sv to linear domain
```

Comment:
These all look good to me. Maybe instead of comments, you can put docstring of what these functions do? This way users know. Otherwise, should probably make it private functions, if this is planned to be used by other classes/functions.

Reply:
Good point about making it a private function.

And like I said, yes we need docstrings for all of these!